engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -1529,6 +1539,8 @@ namespace dmGameSystem
     void SetGravity(void* _world, const Vectormath::Aos::Vector3& gravity)
     {
         CollisionWorld* world = (CollisionWorld*)_world;
+        if (world == 0x0)
```

Comment:
In this case, the world is 0, and you can check it in the `Physics_SetGravity` function.

Reply:
Thx!